app/views/user_registrations/link_github.html.slim

```diff
@@ -0,0 +1,19 @@
+.page-header
+  h3.text-center.green-text Welcome on board! Let's get started
+  
+  p.text-center During the program, you will need a Github repository and account to share your awesome code with your mentor.
+
+  p.text-center Don't have a GitHub account and repo? Check this guide on how to create one! 
+
```

Comment:
When mentee/mentor fills in project details they'd need to provide github link as it's mandatory on update. Without it project details won't be saved. So from the second week we'll be sending reminder to all pairs that haven't provided project details (including github link) and they'll need to enter it anyway so we don't have to have this page.